charts/opentelemetry-demo/values.yaml

```diff
@@ -93,15 +93,15 @@ components:
     # If a different url is desired, set DATABASE_URL here.
     # - name: DATABASE_URL
     #   value:
-    - name: GRPC_PORT
+    - name: FEATURE_FLAG_GRPC_SERVICE_PORT
+      value: '8080'
+    - name: FEATURE_FLAG_SERVICE_PORT
       value: '50053'
```

Comment:
Do these values need to match the ports exposed below?

Reply:
Doh...

Reply:
Should be all set.